ui/src/components/search-result.tsx

```diff
@@ -1,149 +1,177 @@
+/* 
+This file contains the code for the <SearchResult/> Component.
+See "SearchResultProps" on this document for the the properties this component
+uses.
+*/
 
 import React from 'react';
 
+// If not working, run "npm install react-icons"
 import { FaConfluence, FaSlack, FaGoogleDrive } from "react-icons/fa";
 
-import BlueFolder from '../assets/images/blue-folder.svg';
+// This imports the full-color icons
 import Slack from '../assets/images/slack.svg';
 import Confluence from '../assets/images/confluence.svg';
-import GoogleDrive from '../assets/images/google-drive.svg';
+import GoogleDrive from '../assets/images/google-drive.svg'
 
-export interface TextPart{
-    content: string
-    bold: boolean
+export interface TextPart {
+  content: string
+  bold: boolean
 }
-  
+
 export enum ResultType {
-    Docment = "document",
-    Message = "message",
-    Comment = "comment",
-    Person = "person"
-}
-  
-export enum Platform {
-    Confluence = "confluence",
-    Slack = "slack",
-    Drive = "google_drive"
+  Docment = "document",
+  Message = "message",
+  Comment = "comment",
+  Person = "person"
 }
 
-export enum PlatformDisplayName {
-    Confluence = "Confluence",
-    Slack = "Slack",
-    Drive = "Google Drive"
+export enum Platform {
+  Confluence = "confluence",
+  Slack = "slack",
+  GoogleDrive = "google drive"
 }
 
+// These are the attributes of a <SearchResult/> react component.
 export interface SearchResultProps {
-    title: string 
```

Comment:
You are using an indentation of 2 spaces while we use 3 spaces.
Please change it to 3 to avoid making such git changes that aren't really changes of code.
Then reformat file so those changes disappear.